bin/main/org/fundacionjala/at15/katas/README.md

```diff
@@ -0,0 +1,43 @@
+# Katas folder
```

Comment:
Hi @Dru27 , please remove the folder `bin/` and all its contents

Reply:
The observation was taken into account.

Reply:
The observation was taken into account.